source/ways-of-working/new-component/connect-via-vpn.html.md.erb

```diff
@@ -29,5 +29,5 @@ Once this is setup and deployed, you can then make a request to PlatOps in \#pla
 ```` 
```

Comment:
Can we update the example to be: https://github.com/hmcts/aks-sds-deploy/blob/0061d448847a7a16154ae3a23772c4e9a57587f1/components/01-network/21-peering.tf#L82-L104

the other one is in a deprecated repo with a weird name